### PR TITLE
Ot112 8 conversation endpoint

### PR DESCRIPTION
**Pull request by AleYanczuk**

### CODE REVIEW

File: app/controllers/api/v1/conversations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      end
+
+      private
+
+      def conversation_params
+        params.permit(:user1_id, :user2_id, :state)
+      end
+
+      def message_params
+        params.permit(:detail)
```
Comment (JeremasPosta) [inline app/controllers/api/v1/conversations_controller.rb:27]: We do not agree to use `require(:object)` approach on this?

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,9 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+User.create(name:'test',email:'abc@abc.com',password:'123456')
+User.create(name:'test2',email:'abc2@abc.com',password:'123456')
+Conversation.create(user1_id:1,user2_id:2)
+Message.create(detail:'blablabla',user_id:1,conversation_id:1)
+Message.create(detail:'blablablasca',user_id:2,conversation_id:1)
```
Comment (JeremasPosta) [inline db/seeds.rb:13]: Great to add seeds!

**Review by JeremasPosta — APPROVED** [review]
I don't know if can conflict with merge, but the specs shows a few syntax problems to fix,

**Review by fciballero — COMMENTED** [review]
Great Job Ale!!! but at the moment I decided to change the database structure like I did in the sample branch. Maybe we can discuss this in creating a fix in the future.

**Pull request closed without merging**